src/app/components/feed/jh-main-feed.directive.js

```diff
@@ -33,8 +35,44 @@
           Janus.attachMediaStream(video, newVal);
         }
       });
+      
+      $timeout(function(){
+        var video = $('video', element)[0];
+        var mainFeed = document.getElementById('main-feed');
+        video.addEventListener('dblclick', function() {
```

Comment:
No worries. Your code was not wrong, my comment was just an opinion / improvement.